### DISCUSSION TITLE
Bibtex problem in Zettlr 2.0

### DISCUSSION THREAD

**Discussion by mugsuk** [Zettlr/Zettlr#2434] [Q&A] [answered]
In Zettlr 2 the citation/bib function is not working with Bibtex files; it is functional with the json, but when selecting the Citation Database in the Export section of the preferences, if it is a Bibtex file, the following error is generated...

"Cannot read property 'parse' of undefined"

Using Zettlr 1.8.9 the same Bibtex database is read with no problems.

Any ideas on why this may be the case.

Cheers

Murray

**Comment by nathanlesage** [comment, marked as answer]
This was a bug. I've just fixed that – so it's in the next nightly and beta!
